pyrasa/utils/peak_utils.py

```diff
@@ -113,17 +115,17 @@ def get_peak_params(
 
 
 def get_peak_params_sprint(
-    periodic_spectrum,
-    freqs,
-    times,
-    ch_names=[],
-    smooth=True,
-    smoothing_window=1,
-    polyorder=1,
-    cut_spectrum=(1, 40),
+    periodic_spectrum: np.ndarray,
+    freqs: np.ndarray,
+    times: np.ndarray,
+    ch_names: Iterable = [],
```

Comment:
Never use mutable types (lists, dicts, etc) as default arguments. Replace this by a tuple.